<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>133</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find and Replace</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="find_label">
           <property name="text">
            <string>Find what:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="replace_label">
           <property name="text">
            <string>Replace with:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="find_value"/>
         </item>
         <item>
          <widget class="QLineEdit" name="replace_value"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="match_case">
           <property name="text">
            <string>Match case</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="whole_words">
           <property name="text">
            <string>Whole words</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="find_next">
           <property name="text">
            <string>Find next</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="replace">
           <property name="text">
            <string>Replace</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="replace_all">
           <property name="text">
            <string>Replace all</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
